rust/crd/src/listener.rs

```diff
@@ -24,6 +24,9 @@ pub enum KafkaListenerProtocol {
     /// Encrypted and server-authenticated HTTPS connections
     #[strum(serialize = "SSL")]
     Ssl,
+    /// Kerberos authentication
```

Comment:
nit: Please spread the enum variants apart using newlines

Reply:
a88146c